tests/bug-3286-02-linux-evasion/test.yaml

```diff
@@ -17,17 +17,10 @@ checks:
         event_type: alert
         alert.signature_id: 1
   - filter:
-      count: 1
```

Comment:
Why are these checks removed ?

Reply:
Libsuricata has no concept of pcap_cnt because it is not aware of the packet acquisition method. Therefore pcap_cnt is always 0 and not logged. In the above test the alerts are triggered for the same signature and the filter are differentiated by the pcap_cnt. Without the pcap_cnt available I merged the two filters in one and increased the counter to achieve the same result. It is true that the test is slightly less precise as we don't check that we generate an alert at pcap_count 2 and an alert at pcap_count 3 but we only check we have 2 alerts for that specific signature.

Reply:
> It is true that the test is slightly less precise as we don't check that we generate an alert at pcap_count 2 and an alert at pcap_count 3 but we only check we have 2 alerts for that specific signature.

Looks problematic...

> Libsuricata has no concept of pcap_cnt

Could it be added ?

Reply:
Yes it could if we want.
I did not added it because it is a counter specific to pcap replaying. It is not present in other runmodes.
In libsuricata it should be a global packet counter.

Reply:
I think we want to keep the ability to test at which packet in a pcap the alert is triggered